File: API/objects/Moon.php
<?php
    class Moon implements \JsonSerializable
    {
        private
            $phase,
            $description;

        public function get_phase(){
            return $this->phase;
        }

        public function set_phase($phase){
            $this->phase = $phase;
        }

        public function get_description(){
            return $this->description;
        }

        public function set_description($description){
            $this->description = $description;
        }

        public function jsonSerialize() {

            $vars = get_object_vars($this);
            return $vars;
        }
    }
?>

File: CyberLife_Calendar/src/db/pojo/Moon.java
package db.pojo;

/**
 * Moon
 */
public class Moon extends AppointmentDB {

    private int phase;
    private String description;

    public Moon() {
        super("moon");
    }

    /**
     * @return the type
     */
    public int getPhase() {
        return phase;
    }

    /**
     * @param type the type to set
     */
    public void setPhase(int phase) {
        this.phase = phase;
    }

    /**
     * @return the description
     */
    public String getDescription() {
        return description;
    }

    /**
     * @param description the description to set
     */
    public void setDescription(String description) {
        this.description = description;
    }
}